.github/CODEOWNERS

```
@@ -8,4 +8,4 @@
 *               @googleapis/yoshi-python
```

Did you mean to add `@googleapis/cloud-storage-dpe` here as well?
```suggestion
*               @googleapis/yoshi-python @googleapis/cloud-storage-dpe
```

---

Ah yes, thank you